jina/proto/jina.proto

```diff
@@ -19,6 +19,11 @@ message RouteProto {
     StatusProto status = 4; // the status of the execution
 }
 
+message JinaInfoProto {
+    map<string, string> jina = 1; // the jina version running
```

Comment:
Why is there not just one Jina version?

Reply:
it is more general information, I will edit

Reply:
```suggestion
    map<string, string> jina = 1; // information about the system running and package version information including jina
```